core/modules/layout/tests/layout.test

```diff
@@ -3414,3 +3414,39 @@ class LayoutFlexibleTemplateTest extends BackdropWebTestCase {
   }
 
 }
+
+/**
+ * Tests that default layout paths are handled properly.
+ */
+class LayoutPathTest extends BackdropWebTestCase {
+  protected $profile = 'testing';
+  protected $admin_user;
+
+  function setUp() {
```

Comment:
Also here: `public function setUp()`...

Reply:
No other implementation of `setUp()` in that file is declared as a `public function`. Is this one different, or are they all wrong?

Reply:
Nice catch, taking a closer look - LayoutPathTest extends BackdropWebTestCase and there function setUp() is "protected".

Methods declared without any explicit visibility keyword are defined as public.
Most of the time in all the legacy code the visibility isn't explicitly declared. IMO this doesn't have to be addressed here. Only the newly added code should declare visibility.

Reply:
I'm wrong. Actually going from `protected` to `public` is legit

Reply:
But that does trigger a thought: if the base class method was protected, should the derived class method be left protected, just for consistency? I.e., does it NEED to be public?

Reply:
That's a good point. I don't think it needs to be public, but if we are going to change that, it should be done as a separate issue, addressing all tests that extend BackdropWebTestCase

Reply:
So my suggestion is still to declare visibility for *this* setUp() and let further improvements of unrelated legacy code happen in a separate issue. Using "protected" seems more consistent, but "public" doesn't do any harm, either.